shared/profile/index.js

```diff
@@ -13,11 +21,16 @@ class Profile extends Component<void, Props, void> {
     }
   }
 
+  componentDidMount () {
+    this.props.refresh()
+  }
+
   render () {
     return (
       <Render
         showComingSoon={!flags.tabProfileEnabled}
         {...this.props}
+        proofs={this.props.proofs || []}
```

Comment:
the user can be not loaded yet. we have a flow where we init the user in the store so basically this can be empty